src/api.rs

```diff
@@ -298,7 +299,8 @@ impl Default for ChromaSampling {
 }
 
 impl ChromaSampling {
-  // Provides the sampling period in the horizontal and vertical axes.
+
+  /// Provides the sampling period in the horizontal and vertical axes.
```

Comment:
sampling period? i never heard of this way of calling the subsampling amount

Reply:
That's not new in this PR though.

Reply:
the fact that this line's not new to this PR is not an argument to not improve it, but yeah if the naming is used elsewhere it's better to leave it